test/verify.tests.js

```diff
@@ -67,6 +69,90 @@ describe('verify', function() {
     });
   });
 
+  describe('secret or token as callback', function () {
```

Comment:
This function is not properly indented with the rest of the file and doesn't follow the newly introduced .editorconfig

Reply:
I'd like to make this a thing in with #486

Reply:
I agree with @jfromaniello on this one. Since the bad indentation is introduced in this PR there's no need to wait for a different one to fix it.